tests/v2/test_waku_rln_relay.nim

```diff
@@ -666,4 +723,29 @@ suite "Waku rln relay":
     doAssert(badVerifyIsSuccessful)
     # badF=1 means the proof is not verified
     # verification of the bad proof should fail
-    doAssert(badF == 1)
\ No newline at end of file
+    doAssert(badF == 1)
+  
+  test "create a list of membership keys and construct a Merkle tree based on the list":
+    let (list, root) = createMembershipList(100) 
+    debug "created membership key list", list
+    check: 
+      list.len == 200 # two parts for each membership key i.e., one id key and one id commitment
+    debug "the Merkle tree root", root
+    check:
+      root.len == 64
```

Comment:
Why is this 64? Comment above explaining what this is ensuring

Reply:
Sure, done!